codegen.yml

```diff
@@ -1,6 +1,6 @@
 overwrite: true
-schema: "https://raw.githubusercontent.com/GaloyMoney/galoy/main/src/graphql/public/schema.graphql"
-# schema: "../galoy/src/graphql/public/schema.graphql"
+schema: "https://api.staging.galoy.io/graphql"
```

Comment:
we will always have to target a live environment from now on? couldn't we write the graphql somewhere?

Reply:
this is the introspection endpoint. i am not sure how to easily get the raw graphql otherwise because its in the private circles repo (i.e the https://raw.githubusercontent.com... url won't work because they have a `?token` secret appended to the end).

@krtk6160 do you have any ideas on how to write the `supergraph.graphql` somewhere public?

Reply:
@bodymindarts Any thoughts on where we can keep the generated supergraph?
Should we sync it to the mobile repo like we sync the proto files?